app/decorators/classified_listing_decorator.rb

```diff
@@ -0,0 +1,14 @@
+class ClassifiedListingDecorator < ApplicationDecorator
+  DEFAULT_COLOR = "#000000".freeze
```

Comment:
I'm open to other default colors, but this doesn't lool too bad I think:

<img width="1410" alt="Screen Shot 2020-05-05 at 11 25 11" src="https://user-images.githubusercontent.com/47985/81034978-247a8e80-8ec3-11ea-9ed4-64dab18cb3b7.png">

Reply:
I love the DEV listings logo, is that new?

Reply:
That's the social preview template we use to generate the previews for https://twitter.com/devlistings.